velox/functions/prestosql/tests/InPredicateTest.cpp

```diff
@@ -968,7 +970,7 @@ TEST_F(InPredicateTest, arrays) {
 
   auto inValuesWithNulls = makeNullableArrayVector<int32_t>({
       {1},
-      {1, std::nullopt, 2},
+      {1, std::nullopt},
```

Comment:
Out of curiosity, why did you remove the existing value?  It seems like it would cover what you're trying to test (both sides contain null and are a match), why not leave it as is or add this as an additional value?

Reply:
@kevinwilfong Good point. I will restore the {1, std::nullopt, 2} value and add an additional {1, 2, std::nullopt}.